community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java

```diff
@@ -57,7 +57,6 @@
 import org.neo4j.unsafe.impl.batchimport.store.io.IoMonitor;
 
 import static java.lang.System.currentTimeMillis;
-
```

Comment:
Hmm OK, it's just the grouping of imports. So there's apparently a mismatch between the codestyle settings for different IDEs then. I'll have a look at that.